rest-api/db-query.sh

```
@@ -1,25 +1,41 @@
 #!/usr/bin/env bash
 set -euo pipefail
 QUERY="${1:-}"
-LIMIT="${2:-}" # optional
-OFFSET="${3:-}" # optional
-if [ -z "$QUERY" ]; then echo "Usage: $0 <sql-query> [limit] [offset]"; exit 1; fi
+PARAMS_JSON="${2:-}" # optional JSON string for parameters
+if [ -z "$QUERY" ]; then echo "Usage: $0 <sql-query> [params-json]"; exit 1; fi
 source_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 if [ -f "$source_dir/.env" ]; then export $(grep -v '^#' "$source_dir/.env" | xargs); fi
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)